common/character_interactions/00_blackmail_interactions.txt

```diff
@@ -12,7 +12,8 @@
 	is_shown = {
```

Comment:
Something is wrong. I would check `error.log` for `age_` errors.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/113775060-31d67380-9739-11eb-879c-b01dbfdc1504.png)

</details>

Reply:
Oh no not again

Reply:
This is #292